compiler/src/iree/compiler/Codegen/Common/DecomposePackUnPackOps.cpp

```diff
@@ -28,7 +28,14 @@ namespace mlir::iree_compiler {
 
 namespace {
 
+static llvm::cl::opt<bool> enableUnPackDecomposition(
+    "iree-codegen-enable-unpack-decomposition",
+    llvm::cl::desc("Enable Decomposition of tensor.unpack Operation. Disable "
+                   "to vectorize tensor.unpack"),
+    llvm::cl::init(true));
+
```

Comment:
I think it is better to move the flag to `LLVMCPU/Passes.cpp`, and pass it to the pass. I.e., we will need an option (like `tileOuterToOne`) and control it in `LLVMCPU/Passes.cpp`. In this context, we centralize all the options at the same place.

Reply:
Moved to llvmcpu/passes.cpp